ios/RNWebrtcVad.m

```diff
@@ -33,53 +34,86 @@ @implementation RNWebrtcVad {
     self.audioData = nil;
 }
 
-- (dispatch_queue_t)methodQueue
+- (dispatch_queue_t) methodQueue
 {
     return dispatch_get_main_queue();
 }
 
--(instancetype)init {
+-(instancetype) init {
     if (self = [super init]) {
         [AudioInputController sharedInstance].delegate = self;
-        self.audioData = [[NSMutableData alloc] init];
+        cumulativeProcessedSampleLength = 0;
     }
     
     return self;
 }
 
-- (void)dealloc {
+- (void) dealloc {
     voiceDetector = nil;
     self.audioData = nil;
 }
 
 - (void) processSampleData:(NSData *)data
 {
+    if (self.audioData == nil){
+        self.audioData = [[NSMutableData alloc] init];
+    }
+    
     [self.audioData appendData:data];
     
     double sampleRate = [AudioInputController sharedInstance].audioSampleRate;
     
     // Google recommends sending samples (in 10ms, 20, or 30ms) chunk.
     // See: https://github.com/TeamGuilded/react-native-webrtc-vad/blob/master/webrtc/common_audio/vad/include/webrtc_vad.h#L75
     
-    int chunk_size = 0.02 /* seconds/chunk */ * sampleRate * 2 /* bytes/sample */ ; /* bytes/chunk */
+    const double sampleLength = 0.02;
+    
+    cumulativeProcessedSampleLength += sampleLength;
+    int chunkSize = sampleLength /* seconds/chunk */ * sampleRate * 2 /* bytes/sample */ ; /* bytes/chunk */
```

Comment:
maybe nit, but would it make sense to add these units/rates in the variable names themselves instead of comments?